app/routes/admin.py

```
@@ -4791,7 +4791,7 @@ def delete_join_code():
         raise
     except Exception as e:
         db.session.rollback()
-        current_app.logger.error(f"Error deleting join code {join_code}: {e}", exc_info=True)
+        current_app.logger.error(f"Error deleting join code {join_code}: {e}")
```

Ruff (0.16.0)

[error] Undefined name `join_code`

(F821)